src/content/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration.mdx

```diff
@@ -196,7 +196,7 @@ These settings are available for agent configuration. Some settings depend on yo
       </tbody>
     </table>
 
-  Manual override for the path to your local CA bundle. This CA bundle will be used to validate the SSL certificate presented by New Relic's data collection service.
+  Manual override for the path to your local CA bundle. This CA bundle validates the SSL certificate presented by New Relic's data collection service.
```

Comment:
**Suggestion**: Manual override for the path to your local CA bundle. This validates the SSL certificate presented by New Relic's data collection service.